trademapper/js/trademapper.js

```diff
@@ -320,6 +312,8 @@ function($, d3, analytics, arrows, csv, filterform, mapper, route, yearslider,
 		this.addChangeFilterSpecLink(elFilterSpecChange);
 		// finally ensure the tab is now available
 		document.querySelector('li[role=filters]').style.display = "block";
+        document.querySelector('li[role=export]').style.display = "block";
+        document.querySelector('li[role=options]').style.display = "block";
```

Comment:
This will now show the options tab, which wasn't visible before. I think it should stay hidden as it's not complete.

Reply:
sure thing, please remove
```suggestion
        document.querySelector('li[role=options]').style.display = "block";
```

Reply:
I will remove and push with css file